commit 5663d4d3b831dcaaf02ca43a22fbdedc37653022 (refs/changes/20/4020/4)
Author: sjorge <sjorge@blackdot.be>
Date:   2018-06-06T11:34:38+00:00 (1 year, 4 months ago)
    
    OS-6968 Increase size of root by 2M
    Reviewed by: John Levon <john.levon@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/tools/build_live b/tools/build_live
index 36f87177..340e0504 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -911,7 +911,7 @@ bi_setup_work_dir
 # Create and mount the "/" (root) ramdisk image:
 #
 bi_file_root="$bi_tmpdir/root.image"
-bi_root_size=276000
+bi_root_size=278000
 if strings "$bi_wsroot/proto/kernel/amd64/genunix" |
     grep "DEBUG enabled" >/dev/null; then
 	#
